editoast/src/views/stdcm_search_environment.rs

```diff
@@ -250,27 +284,19 @@ pub mod tests {
             .json_into::<StdcmSearchEnvironment>();
 
         // THEN
-        assert_eq!(
-            stdcm_search_env,
-            StdcmSearchEnvironment {
-                id: stdcm_search_env.id,
-                infra_id: infra.id,
-                electrical_profile_set_id: Some(electrical_profile_set.id),
-                work_schedule_group_id: Some(work_schedule_group.id),
-                temporary_speed_limit_group_id: Some(temporary_speed_limit_group.id),
-                timetable_id: timetable.id,
-                search_window_begin: begin,
-                search_window_end: end,
-            }
-        );
+        assert_eq!(stdcm_search_env.enabled_from, enabled_from);
+        assert_eq!(stdcm_search_env.enabled_to, enabled_to);
     }
 
     #[rstest]
     async fn retrieve_stdcm_search_env_not_found() {
+        use editoast_models::tables::stdcm_search_environment::dsl::*;
         // GIVEN
         let app = TestAppBuilder::default_app();
-
-        let _ = StdcmSearchEnvironment::delete_all(&mut app.db_pool().get_ok()).await;
+        diesel::delete(stdcm_search_environment)
```

Comment:
Isn't using diesel in the `views` crate frowned upon these days ?

Reply:
Yeah true. I'll fix it